base/complex.jl

```diff
@@ -19,8 +19,13 @@ Complex(x::Real) = Complex(x, zero(x))
     im
 
 The imaginary unit.
+
+# Example
+```jldoctest
+julia> im * im
+-1 + 0im
```

Comment:
Yes, a missing closing backtick has the effect that the doctest is not run at all.